lib/scala/tree/src/test/scala/io/github/tetherlessworld/twxplore/lib/tree/TestData.scala

```diff
@@ -0,0 +1,19 @@
+package io.github.tetherlessworld.twxplore.lib.tree
+
+import io.github.tetherlessworld.twxplore.lib.geo.models.domain._
+
+import scala.collection.mutable
+
+object TestData {
+  private val testData = TreeDataCsvTransformer()
+  testData.parseCSV("./lib/scala/tree/src/test/resources/test_treedata.csv")
```

Comment:
This assumes a current working directory. Use 
https://stackoverflow.com/questions/27360977/how-to-read-files-from-resources-folder-in-scala